notebooks/image_models/labs/autoencoder_anomaly_detection.ipynb

```diff
@@ -0,0 +1,750 @@
+{
```

Comment:
<p>I would add an exercise for the two cells above. For instance:</p><p> </p><p><strong>Exercise:</strong> In the cell below, use <code>build_autoencoder</code> to instantiate an auto-encoder model. Then write the code to compile and train it.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/215/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>